### PR TITLE
add details about pooler auth

### PR DESCRIPTION
**Pull request by sebastianwebber**
# Description

This PR extends the connection pooler docs, adding details about how to use custom users.

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Comment by Loquacity** [comment]
Preview is up: https://docs-dev.timescale.com/docs-sebastian-update-pool-user-settings/use-timescale/sebastian-update-pool-user-settings/services/connection-pooling/#user-authentication

File: use-timescale/services/connection-pooling.md [hunk 1]
```diff
@@ -28,6 +28,63 @@ performance because the pooler open the connections in advance,  allowing the
 application to open many short-lived connections, while the database opens few,
 long-lived connections.
 
+## User authentication
+
+By default, the pooler passes through authentication to the database, so you can
+keep using any custom users you already have set up without further
+configuration. You can also continue using the `tsdbadmin` user, if that is your
+preferred method. However, for some cases, you might need to add custom
+configurations for a pooler user, such as `statement_timeout`.
```
Comment (Rajakavitha1) [inline use-timescale/services/connection-pooling.md:37]:
```suggestion
By default, the poolers have authentication to the database, so you can use any
custom users you already have set up without further configuration. You can continue
using the `tsdbadmin` user if that is your preferred method. However, you might need
to add custom configurations for some cases such as `statement_timeout` for a pooler user.
```

File: use-timescale/services/connection-pooling.md [hunk 2]
```diff
@@ -28,6 +28,63 @@ performance because the pooler open the connections in advance,  allowing the
...
+
+<Procedure>
+
+### Creating a new user with custom settings
+
+1.  Log in to your database as the `tsdbadmin` user, and create a new role:
```
Comment (Rajakavitha1) [inline use-timescale/services/connection-pooling.md:43]:
```suggestion
1.  Log in to your database as the `tsdbadmin` user, and create a new role named `<MY_APP>` with the password as `<PASSWORD>`:
```

File: use-timescale/services/connection-pooling.md [hunk 3]
```diff
@@ -28,6 +28,63 @@ performance because the pooler open the connections in advance,  allowing the
...
+
+    ```sql
+    CREATE ROLE my_app LOGIN PASSWORD '<NEW_PASSWORD>';
+    ```
+
+1.  Alter the role to change the setting for this user. This example changes the
```
Comment (Rajakavitha1) [inline use-timescale/services/connection-pooling.md:49]:
```suggestion
1.  To change the `statement_timeout` settings to 2 seconds for this user:

File: use-timescale/services/connection-pooling.md [hunk 4]
```diff
@@ -28,6 +28,63 @@ performance because the pooler open the connections in advance,  allowing the
...
+   `statement_timeout` to 2 seconds:
```
Comment (Rajakavitha1) [inline use-timescale/services/connection-pooling.md:50]:
```suggestion
   
```

File: use-timescale/services/connection-pooling.md [hunk 5]
```diff
@@ -28,6 +28,63 @@ performance because the pooler open the connections in advance,  allowing the
...
+
+    ```sql
+    ALTER ROLE my_app SET statement_timeout TO '2s';
+    ```
+
+1.  Connect on the pooler with the new user:
```
Comment (Rajakavitha1) [inline use-timescale/services/connection-pooling.md:56]:
```suggestion
1.  In a new terminal window, connect on the pooler with the new user `<MY_APP>`:
```

File: use-timescale/services/connection-pooling.md [hunk 6]
```diff
@@ -28,6 +28,63 @@ performance because the pooler open the connections in advance,  allowing the
...
+
+    The output looks something like this:
+
+    <CodeBlock canCopy={false}
+    showLineNumbers={true}
+    children={`
+    psql (15.3 (Homebrew), server 15.4 (Ubuntu 15.4-1.pgdg22.04+1))
+    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
+    Type "help" for help.
+    tsdb=> select current_user;
```
Comment (Rajakavitha1) [inline use-timescale/services/connection-pooling.md:70]: This is a new step to check if you looked in as '<MY_APP>':

File: use-timescale/services/connection-pooling.md [hunk 7]
```diff
@@ -28,6 +28,63 @@ performance because the pooler open the connections in advance,  allowing the
...
+    ┌──────────────┐
+    │ current_user │
+    ├──────────────┤
+    │ my_app       │
+    └──────────────┘
+    (1 row)
+    tsdb=> show statement_timeout;
```
Comment (Rajakavitha1) [inline use-timescale/services/connection-pooling.md:77]: This is another step. To check the `statement_timeout` setting soft `<MY_APP>` user:

**Review by Rajakavitha1 — CHANGES_REQUESTED** [review]

**Comment by Rajakavitha1** [comment]
Thank you for the PR @sebastianwebber . I tested and validated the steps they work as documented. 
```
rajakavithakodhandapani@Rajakavithas-MacBook-Pro-2 ~ % PGPASSWORD=test123 psql "postgres://my_app@cpda2ec06w.oudd3s6gsr.tsdb.cloud.timescale.com:30080/tsdb?sslmode=require"
psql (14.7 (Homebrew), server 15.4 (Ubuntu 15.4-1.pgdg22.04+1))
WARNING: psql major version 14, server major version 15.
         Some psql features might not work.
SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
Type "help" for help.

tsdb=> select current_user;
 current_user 
--------------
 my_app
(1 row)

tsdb=> show statement_timeout;
 statement_timeout 
-------------------
 2s
(1 row)

tsdb=> 
```

**Review by Rajakavitha1 — APPROVED** [review]
LGTM

**Review by ggodeke — APPROVED** [review]
Looks great -- thanks Sebastian!

**Comment by ggodeke** [comment]
I am not sure how to fix these errors :oh-no:

**Comment by jonatas** [comment]
Hey @sebastianwebber, looks like we have a error here.

> {"message": "[Google.Units] Put a nonbreaking space between the number and the unit in '2s'.", "location": {"path": "use-timescale/services/connection-pooling.md", "range": {"start": {"line": 91, "column": 7}}}, "severity": "ERROR"}



Probably the change here:

```diff
+    ALTER ROLE my_app SET statement_timeout TO '2s';
```

Probably we should find a way to ignore this case on prose config file.

**Pull request merged**